src/awsrun/runner.py

```diff
@@ -835,24 +865,25 @@ class AccountRunner:
     execute method, or exceptions raised, are made available to the command.
     """
 
-    def __init__(self, session_provider, max_workers=10):
+    def __init__(self, session_provider, max_workers=10, context=None):
         if not isinstance(session_provider, SessionProvider):
             raise TypeError(
                 f"'{session_provider}' must be a subclass of awsrun.session.SessionProvider"
             )
 
         self.session_provider = session_provider
         self.max_workers = max_workers
+        self.context = context
 
-    def run(self, cmd, accounts, key=lambda x: x):
+    def run(self, cmd, accounts, key=lambda x: x, context=None):
```

Comment:
You shouldn't need to adjust the `run` method as you saved the context as an instance variable during initialization.